### ISSUE TITLE
Status CatlightNotFound

### ISSUE THREAD

**Issue by ScottLangham** [matthiaslischka/CatlightApi#2]
I kept getting CatlightNotFound reported as the status even though CatLight was running ok and its dashboard displayed the correct status.

There was more than one port number listed in the CatLight log files. This api tries to scrape the port number by reading the log files and I believe it must have been reading an out of date port number.

I managed to work around the issue by deleting all the log files from here: "c:\users\<myUser>\AppData\Roaming\Catlight\Logs"
And then restarted CatLight to create a fresh log file.

**Comment by matthiaslischka** [comment]
Hi,
it should work just fine with more port numbers in the same logfile. I'm currently on holidays but will look into it very soon. Thanks for the feedback! If this happens again would you mind providing the logfiles in question for analysis?

BR

**Comment by ScottLangham** [comment]
Yes, unfortunately I’d deleted the log files.. will get them if it happens again.
